### PR TITLE
VEN-1103 | Update profile contact information

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description :sparkles:

* Edit customer contact information

### Closes :no_good_woman:

* [VEN-1103](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1103): Edit customer contact information

### Manual testing :construction_worker_man:

* Log in
* Create profile
* Link Venepaikat service to profile
* Edit profile information

### CODE REVIEW

File: src/features/profile/contactInfo/EditContactInfo.tsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          <h1>{t('page.profile.contact.edit')}</h1>
+          <div className="vene-edit-contact-info__section">
+            <FormField id="firstName" label={t('page.profile.contact.first_name')} />
+            <FormField id="lastName" label={t('page.profile.contact.last_name')} />
+          </div>
+          <div className="vene-edit-contact-info__section">
+            <FormField id="address" label={t('page.profile.contact.address')} />
+            <FormField id="postalCode" label={t('page.profile.contact.postal_code')} />
+            <div className="vene-edit-contact-info__field">
+              <Field name="municipality">
```
Comment (Nizar-Rahme) [inline src/features/profile/contactInfo/EditContactInfo.tsx:64]: I wonder if these names conflict with other names which are used by React Final Form in Redux.
If so, I suggest prefixing the names and ids with the component name for example.

File: src/features/profile/contactInfo/ContactInfo.tsx [hunk 2]
```diff
@@ -1,34 +1,36 @@
...
-  phoneNumber,
-  emailAddress,
-  customerGroup,
-  language,
-}: ContactInfoProps) => {
+const ContactInfo = ({ contactInfo, handleEdit }: ContactInfoProps) => {
   const { t } = useTranslation();
 
+  if (!contactInfo) {
+    return <div>Sinulla ei ole vielä profiilia. Luo uusi profiili tästä.</div>;
```
Comment (Nizar-Rahme) [inline src/features/profile/contactInfo/ContactInfo.tsx:17]: Do we have translations for this statement?

**Review by Nizar-Rahme — CHANGES_REQUESTED** [review]
A few comments to address; otherwise, it looks good!

File: src/locales/sv.json [hunk 3]
```diff
@@ -35,7 +35,12 @@
     "waste_collection": "Avfallshantering",
     "water": "Vatten",
     "width": "Bredd",
-    "year": "År"
+    "year": "År",
+    "language": {
+      "FINNISH": "finska",
+      "ENGLISH": "engelska",
+      "SWEDISH": "svenska"
```
[the remark this answers is not in the document]
Comment (TatuArvela) [reply src/locales/sv.json:42]: In Swedish, the first letters of languages are written in lowercase, and I went with that

File: src/features/profile/contactInfo/EditContactInfo.tsx (hunk 1 above)
Comment (TatuArvela) [reply src/features/profile/contactInfo/EditContactInfo.tsx:64]: They shouldn't be, but I'll have a look

File: src/features/profile/contactInfo/ContactInfo.tsx (hunk 2 above)
Comment (TatuArvela) [reply src/features/profile/contactInfo/ContactInfo.tsx:17]: Not yet 🤔

File: src/features/profile/contactInfo/EditContactInfo.tsx (hunk 1 above)
Comment (Nizar-Rahme) [reply src/features/profile/contactInfo/EditContactInfo.tsx:64]: If they don't, this should be fine. 
Approving

**Review by Nizar-Rahme — APPROVED** [review]

File: src/features/profile/contactInfo/EditContactInfo.tsx (hunk 1 above)
Comment (TatuArvela) [reply src/features/profile/contactInfo/EditContactInfo.tsx:64]: This seems fine as it is

File: src/features/profile/contactInfo/ContactInfo.tsx (hunk 2 above)
Comment (TatuArvela) [reply src/features/profile/contactInfo/ContactInfo.tsx]: I moved this to Container level and changed it to English. There needs to be a separate profile creation form. I can start it today, but someone probably has to pick up the work later

**Pull request merged**